pkg/api/meta/help_test.go

```diff
@@ -114,7 +115,10 @@ type fakePtrInterfaceList struct {
 	Items *[]runtime.Object
 }
 
-func (f fakePtrInterfaceList) IsAnAPIObject() {}
+func (obj fakePtrInterfaceList) SetGroupVersionKind(gvk *unversioned.GroupVersionKind) {}
+func (obj fakePtrInterfaceList) GroupVersionKind() *unversioned.GroupVersionKind {
+	return nil
```

Comment:
I'll update the GoDoc to make that clear.

On Thu, Dec 10, 2015 at 9:24 AM, Clayton Coleman <ccoleman@redhat.com>
wrote:

> That's not the interface it's providing.  It's providing "allow this
> object, which should be serialized, to have its values set".  Supporting
> the interface does not mean that setting those values does anything.
>
> On Thu, Dec 10, 2015 at 7:45 AM, David Eads <notifications@github.com>
> wrote:
>
>> In pkg/api/meta/help_test.go
>> <https://github.com/kubernetes/kubernetes/pull/18473#discussion_r47221593>
>> :
>>
>> > @@ -114,7 +115,10 @@ type fakePtrInterfaceList struct {
>> >  	Items *[]runtime.Object
>> >  }
>> >
>> > -func (f fakePtrInterfaceList) IsAnAPIObject() {}
>> > +func (obj fakePtrInterfaceList) SetGroupVersionKind(gvk *unversioned.GroupVersionKind) {}
>> > +func (obj fakePtrInterfaceList) GroupVersionKind() *unversioned.GroupVersionKind {
>> > +	return nil
>>
>> I would rather have something here, even if the group is just
>> "api.meta.test"
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/kubernetes/kubernetes/pull/18473/files#r47221593>.
>>
>
>